docs/cub/developer_overview.rst

```diff
@@ -612,8 +620,11 @@ An agent policy could look like this:
 
 It's typically a collection of configuration values for the kernel launch configuration,
 work distribution setting, load and store algorithms to use, as well as load instruction cache modifiers.
+A CUB algorithm can have multiple agents and thus use multiple agent policies.
 
-Finally, the tuning policy hub looks like:
+Since the device code of CUB algorithms is compiled for each PTX version, a different agent policy may be used.
+Therefore, all agent policies of a CUB algorithm, called a policy, may be replicated for several minimum PTX versions.
+A chained collection of such policies finally forms a policy hub:
```

Comment:
I am still struggling a bit to understand some relationships between the following:

- Algorithm
- Agent
- Agent Policy
- PTX version
- "Policy"
- PolicyHub

The only thing I think I'm sure of is that a PolicyHub is a collection of Policies.

Reply:
More like a dispatcher to a collection of policies I'd say

Reply:
We should probably define the terms somewhere once we have settled on their names. PTX version is actually SM version/compute capability and has nothing to do with PTX. It's just historic I think. Agent policy I derived from the names in the source code. PolicyHub was a termin Georgii coined at some point (it was previously a PtxPlan in Thrust). And the "Policy" is probably what we started with when customizing a CUB algorithm.

Reply:
Let's follow up later: #3976